man/timesyncd.conf.xml

```diff
@@ -101,6 +101,36 @@
         is used instead.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MinPollIntervalSec=</varname></term>
+        <term><varname>MaxPollIntervalSec=</varname></term>
+        <listitem><para>The minimum and maximum poll intervals for NTP messages.
+        Take time (in seconds). <varname>MinPollIntervalSec=</varname> must be larger than 16 sec.
```

Comment:
The code checks for "no smaller than 16 s".

Please either use "s" (the SI unit) or "second", not the "sec" abomination.